scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+@import "colors";
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  border-top: 6px solid $cerulean;
+  box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
+  display: flex;
+  height: 70px;
```

Comment:
it would be a good idea to store this in a variable, you will use it on another views